source/_integrations/magichome.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: "MagicHome"
+description: "Instructions on how to setup the MagicHome hub within Home Assistant."
+logo: magichome.png
```

Comment:
Please open a PR to add the brand logo's on our brands repository:

<https://github.com/home-assistant/brands>

Remove the `logo` property after that and link the PR @ the Brands repository to this PR and the parent PR.

Thanks! 👍

Reply:
OK！